src/components/AddressSearch/types.ts

```diff
@@ -0,0 +1,96 @@
+import type {RefObject} from 'react';
+import type {NativeSyntheticEvent, StyleProp, TextInputFocusEventData, View, ViewStyle} from 'react-native';
+import type {Place} from 'react-native-google-places-autocomplete';
+import type Locale from '@src/types/onyx/Locale';
+
+type CurrentLocationButtonProps = {
+    /** Callback that is called when the button is clicked */
+    onPress: () => void;
```

Comment:
```suggestion
    onPress?: () => void;
```

This is optional, right?

Reply:
Right, corrected